Packs/CommonScripts/ReleaseNotes/1_10_44.md

```diff
@@ -0,0 +1,4 @@
+
+#### Scripts
+##### New: GenerateAsBuilt
+- Generate an as built document, as HTML, based on the running XSOAR instance. Requires an instance of the Demisto API integration configured. (Available from Cortex XSOAR 6.0.0).
```

Comment:
Add here also the update docker image note

Reply:
Fixed.